src/main/java/net/imglib2/roi/boundary/Boundary.java

```diff
@@ -0,0 +1,296 @@
+package net.imglib2.roi.boundary;
+
+import static net.imglib2.roi.boundary.Boundary.StructuringElement.FOUR_CONNECTED;
```

Comment:
not sure. I was thinking about reusing the `CCA` as well, but `CCA` attaches more to its `StructuringElement` (factories), therefor it didn't work. It might sense to restructure it, though. Next time it comes up, I would say...